From: Russ Allbery <rra@debian.org>
Date: Sun, 17 Feb 2008 22:17:59 -0800
Subject: Minor man page fixes

Escape hyphens properly, add a better man page header, fix the
placement of punctuation around font changes, and fix a spelling
error.
---
 src/tf.1.nroffman | 34 +++++++++++++++++-----------------
 1 file changed, 17 insertions(+), 17 deletions(-)

diff --git a/src/tf.1.nroffman b/src/tf.1.nroffman
index e8daaca..90691db 100644
--- a/src/tf.1.nroffman
+++ b/src/tf.1.nroffman
@@ -1,11 +1,11 @@
 .\" $Id: tf.1.nroffman,v 34000.8 2007/01/13 23:12:39 kkeys Exp $
-.TH TF 1 LOCAL
+.TH TF 1 "2007-01-13" "TinyFugue 5.0" "TinyFugue"
 .SH NAME
 tf \- TinyFugue, a MUD client
 .SH SYNOPSIS
-.B "tf [-f\fIfile\fB] [-lnq] [\fIworld\fB]"
+.B "tf [\-f\fIfile\fB] [\-lnq] [\fIworld\fB]"
 .br
-.B "tf [-f\fIfile\fB] \fIhost\fB \fIport\fB"
+.B "tf [\-f\fIfile\fB] \fIhost\fB \fIport\fB"
 .SH DESCRIPTION
 \fITinyFugue\fR (also known as "Fugue" or "TF") is a line-based client
 designed for connecting to MUD servers (note: LP, DIKU, and other servers
@@ -48,15 +48,15 @@ world and try to connect to it.
 The \fBhost\fR may be an IP number or regular name format.
 .PP
 OPTIONS
-.IP "-f\fIfile\fR"
+.IP "\-f\fIfile\fR"
 Load \fIfile\fR instead of $HOME/.tfrc at startup.
-.IP "-f"
+.IP "\-f"
 Do not load any personal configuration file.  The library will still be loaded.
-.IP "-l"
+.IP "\-l"
 Disable automatic login.
-.IP "-n"
+.IP "\-n"
 Do not connect to any world at startup.
-.IP "-q"
+.IP "\-q"
 Quiet login (overrides %{quiet} flag).
 
 .SH FEATURES
@@ -109,7 +109,7 @@ Log a session to a file.
 .sp
 Separate LP and Diku style prompts from normal output.
 .sp
-Page output using a --More-- prompt.
+Page output using a \-\-More\-\- prompt.
 .sp
 Recall previously received text.
 
@@ -120,14 +120,14 @@ configuration file "stdlib.tf", located in TFLIBDIR.
 TFLIBDIR is defined when \fITF\fR is installed, and is often
 /usr/local/lib/tf.lib, or under the home directory of the installer.
 This library contains many macros
-and definitions essential to the correct operation of \fITF.\fR
+and definitions essential to the correct operation of \fITF\fR.
 
 Next, \fITF\fR will attempt to read your personal configuration
 file, $HOME/.tfrc, in which you can put any \fITF\fR commands you
 want executed automatically at startup.  Two of the most useful
 commands to use in a \fITF\fR configuration file are /addworld and /load.
 
-For backward compatability, \fITF\fR will also try
+For backward compatibility, \fITF\fR will also try
 to read the \fITinyTalk\fR configuration file.
 Its name defautls to $HOME/.tinytalk, or can be defined by the TINYTALK
 environment variable.
@@ -140,19 +140,19 @@ the first Tinyclient with any great number of features, including
 hiliting and suppression of text, simple triggers, and separating input and
 output on the screen.
 Leo Plotkin (Grod) made rather extensive modifications to \fITinytalk\fR
-to produce \fITinywar,\fR which was plagued with some
+to produce \fITinywar\fR, which was plagued with some
 serious bugs and was never officially released (the phrase "Tinywar
 doesn't exist" is often quoted), and is now an unsupported client.
 \fITF\fR began when Greg Hudson (Explorer_Bob) merged many of the new
-features of \fITinywar\fR back into \fITinyTalk,\fR
+features of \fITinywar\fR back into \fITinyTalk\fR,
 and added many new features of his own, most notably the split screen.
 Some of the code in Greg's releases was contributed by Leo Plotkin.
-After Greg moved on to \fIVaporTalk,\fR Ken Keys (Hawkeye) took over design
+After Greg moved on to \fIVaporTalk\fR, Ken Keys (Hawkeye) took over design
 and maintenance of \fITF\fR in July 1991, and continues to make improvements
 in features and performance.
 .PP
 The code size of \fITF\fR has surpassed 300K (unstripped),
-and is signifigantly larger than \fITinytalk.\fR
+and is signifigantly larger than \fITinytalk\fR.
 It is, in fact, more than three times the size of a number of existing servers.
 As of version 3.0, it has 66 builtin commands and 57 library commands,
 each documented in the helpfile.
@@ -225,8 +225,8 @@ default macro library.
 .PP
 The .tinytalk file may not be supported in the future; use .tfrc instead.
 .PP
-The '-' command line option in versions prior to 2.0 is no longer supported,
-and has been replaced with '-l'.
+The '\-' command line option in versions prior to 2.0 is no longer supported,
+and has been replaced with '\-l'.
 
 .SH BUGS
 .PP
